thrifty-kotlin-codegen/src/main/kotlin/com/microsoft/thrifty/kgen/KotlinCodeGenerator.kt

```diff
@@ -149,6 +151,7 @@ class KotlinCodeGenerator(
     private var emitJvmName: Boolean = false
     private var emitJvmStatic: Boolean = false
     private var emitBigEnums: Boolean = false
+    private var emitFileComment: Boolean = true
```

Comment:
Personally I'd prefer this to default to false, _if only_ to minimize the disruption to others.  AFAIK most people using thrifty are still manually checking in the generated code.

Reply:
@benjamin-bader I don't have a strong opinion either way, but it does default to true for Java sources, which means the default will change depending on whether you generate Java or Kotlin. I will defer to you.

Reply:
You make a good point.  I'd forgotten entirely about the Java code-generator, despite directly referring to it.  Shows where our focus is, I suppose.  Let's keep it on by default, then.